themes/scss/map/dashboard-info.scss

```diff
@@ -157,15 +155,15 @@
 }
 .CDB-Dashboard-infoSnapshotsListTitle {
   margin-bottom: 8px;
-  font-size: $sFontSize-smaller;
+  font-size: $sFontSize-small;
```

Comment:
Name of variable `sFontSize-small` should be written in all lowercase letters with hyphens instead of underscores